pkg/connector/helpers.go

```diff
@@ -62,6 +62,20 @@ func formatEntitlementID(resource *v2.Resource, privName string, grant bool) str
 	}
 }
 
+func parseEntitlementID(id string) (string, string, bool, error) {
```

Comment:
tiny, unrelated to this PR: wish we had better structure for this in the general baton sdk --  we re-use ID -> splitting alot -- or maybe it should be a way to use a Protobuf as an ID?